### ISSUE TITLE
CUPS En pause "printer error: No paper

### ISSUE THREAD

**Issue by HBTphoto** [OpenPrinting/cups#278]
Hello,
Everything work well until there is no paper in my canon Selphy CP1000

The state of the printer go to “En pause – “Printer error: No Paper (03)”

Even if load paper, turn Off then On the printer, the state in CUPS don’t change and there no more print
I need to manually click on “start the printer” in CUPS interface, then printer is working well.
`sudo cupsenable Canon_CP1000` work also

I read on the web that “ErrorPolicy retry-job” could solve this problem, but this rule seems to be already set by default in my CUPS version 2.2.10
![IMG_20211025_220916 (1)](https://user-images.githubusercontent.com/93097313/138774662-47659f29-26cb-4b22-a06b-42baa2e1b069.jpg)

Is somebody had the same problem?
Thank for your help

**Comment by zdohnal** [comment]
Thank you for the logs!

JFYI it is better to start catching logs right before you trigger the issue you have a problem with and cancel it right after - then the file contains more relevant logs and it is easier to investigate it. In the attached file I see cupsd start and web ui requests as well, so it is an indicator that log capture was started earlier and canceled later than expected.

From the logs I can see the reason why the queue is stopped and you need to enabled it to run it again. There are two things which need to be in place for ```retry-job``` to be activated:

- the job has to be in processing (IPP_JOB_PROCESSING) or completed (IPP_JOB_COMPLETED) states
- the backend has to return failed (CUPS_BACKEND_FAILED) or retry (CUPS_BACKEND_RETRY) states

For your job, the job is in the processing state, but the gutenprint backend itself returns CUPS_BACKEND_STOP, which disables the print queue in CUPS.

To sum it up, the queue disabled correctly in CUPS because it is told to do so. You can contact Gutenprint developers on their mailing list - gimp-print-devel@lists.sourceforge.net - to see whether the error code from the backend is correct or if it is a bug.

**Comment by HBTphoto** [comment]
Ok, thank, i send the mail, we will see.
